tools/ubuntu-setup/all.sh

```diff
@@ -17,8 +18,14 @@ echo "*** install scala"
 "$SCRIPTDIR/scala.sh"
```

Comment:
Go ahead and remove scala from all.sh, it's not need it gradle takes care of this.